src/Engine/mm7_data.h

```diff
@@ -182,13 +183,13 @@ extern unsigned int uIconID_TurnStart;
 extern int dword_50C994;
 extern int dword_50C998_turnbased_icon_1A;
 extern int uSpriteID_Spell11;  // idb
-extern bool _50C9A0_IsEnchantingInProgress;
-extern int _50C9A8_item_enchantment_timer;
-extern int _50C9D0_AfterEnchClickEventId;
-extern int _50C9D4_AfterEnchClickEventSecondParam;
-extern int _50C9D8_AfterEnchClickEventTimeout;
-extern int dword_50C9DC;
-extern struct NPCData *ptr_50C9E0;
+extern bool IsEnchantingInProgress;
```

Comment:
Since you're touching these and have a good grasp of what they do -- please add a short comment to each